### PR TITLE
feat(api): Finalize mfa support

### PR DESCRIPTION
**Pull request by hamid-yg**
# Description

Fix MFA support (Back and Front)

# Fixes

Specify which issue this PR targets, like so:

- fixes #386 

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it.

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added @vrn-sh/members to the reviewers, or specific members of the team
- [x] I have added the needed labels
- [x] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it.

### CODE REVIEW

**Comment by hamid-yg** [comment]
I have just one issue to fix on this PR. The checkbox isn't checked even when the MFA is enable

**Review by djnnvx — COMMENTED** [review]
back-end fixes look OK @claraxuxu

File: web/src/pages/Settings/Settings-pages/securityUser.tsx [hunk 1]
```diff
@@ -49,10 +52,122 @@ export default function SecurityUser() {
...
+    const [isCodeIncorrect, setIsCodeIncorrect] = useState(false); // État pour suivre si le code est incorrect
+
+    const handleVerifyCode = () => {
+        axios.post(`${config.apiUrl}/mfa?mfa_code=${mfaCode}`, {}, {
+            headers: {
+                'Content-type': 'application/json',
+                'Authorization': `Token ${Cookies.get('Token')}`,
+            },
+        }).then((data) => {
+            console.log(data)
```
Comment (claraxuxu) [inline web/src/pages/Settings/Settings-pages/securityUser.tsx]: need to delete console.log

**Review by claraxuxu — APPROVED** [review]
all good for front

**Pull request merged**